rfcs/070-concepts-api-changes/README.md

```diff
@@ -0,0 +1,324 @@
+The screenshot below shows all concept metadata exposed on this page type:
+
+> <img src="img/example_subject_theme_page.png" alt="example_subject_theme_page" width="600"/>
+
+This page only has one additional piece of information not shown on other theme pages (refer to tables above for all other fields):
+
+
+| Field name   | Details                                                                            |
+|--------------|------------------------------------------------------------------------------------|
+| narrowerThan | Stored in the graph as SourceConceptNarrowerThan and SourceConceptHasParent edges. |
+
+
+## Works and images shown on theme pages
+
+Like current theme pages, new theme page designs include sections listing works and images related to the given theme.
+However, new theme pages expand the number of available tabs, providing more extensive filtering options.
+
+Current theme pages populate their Works tabs by making requests to the catalogue API, specifying the canonical ID of
+the given theme as a filter. New theme pages will use the same approach, applying additional filters as needed.
+
+### Contributor theme pages
+
+New Contributor theme pages include tabs for filtering all works linked to the given Person/Agent/Organisation by 'format' (also known as `workType`): 
+
+> <img src="img/works_tabs_person.png" alt="works_tabs_person" width="600"/>
+
+These tabs will be populated by applying the relevant `workType` filter. For example, to obtain all books about John Thomson,
+a search request will be made to the catalogue API, specifying a `subjects` filter (including IDs of all 'merged' John Thomson concepts)
+and a `workType` filter (including the ID of the 'Books' `workType`).
+
+### Subject theme pages
+
+New subject theme page designs include additional tabs listing works linked to sub topics (child themes) of the given theme:
+
+> <img src="img/works_tabs_concept.png" alt="works_tabs_concept" width="600"/>
+
+To retrieve these works from the catalogue API, we need the *single concept endpoint* to return a list of relevant narrower topics
+(in addition to the fields listed in the tables above):
+
+| Field name  | Details                                                                                                                                                                                                                                          |
+|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| broaderThan | Stored in the graph as SourceConceptNarrowerThan (edge direction reversed) and SourceConceptHasParent (edge direction reversed) edges. |
+
+We can then make separate API calls to the catalogue API to retrieve a list of works for each sub topic.
```

Comment:
I am interested in this; in `/search`, we also have tabs and in all designs, each tab has the X of results as a suffix. Because this means having to do multiple queries on every keyword change (and ignoring the added difficulty of filtering), we've always pushed back on it. I'd be interested in seeing how it's dealt with on your end and will keep an eye out 👀